### ISSUE TITLE
accounts balance displays incorrect results for prolonged periods

### ISSUE THREAD

**Issue by msa6867** [iron-fish/ironfish#1637]
### What happened?

Incorrect amount displayed in accounts:balance following deposits; seems to eventually revert to correct balance after several deposits.  Note that reversion to correct blanace does not seem to correlate well to the mere passage of time, as the incorrect value will persist to display after long periods of time if no further transactions are made.

Example log output from acounts balance queries shown below.  Note that each query occurs after deposits made in separate command line, always waiting for deposits to settle and appear in block explorer before doing a new balance query.  Any duplicate account balance shown is the result of placing a new query after approx 1 hour delay with no intervening deposit.  

To summarize this sequence I show displayed value correct value below

displayed                             correct
5.18354554                          5.18354554
5.16682201                          5.08354553
5.16682201                          5.08354553 (re-queried after long while to see if value would change)
5.06682200                          4.98354552
4.78354550                          4.78354550 (back in sync.  Note tested behavior of 2 simultaneous deposits from different CLI - no problems there)
4.68354549                          4.68354549
4.62973164                          4.58354548
4.62973164                          4.58354548 (re-queried after long while to see if value would change)
4.57706473                          4.48354547
4.48509192                          4.38354546
4.28354545                          4.28354545 (back in sync)

### Version

v0.1.36

### Debug output

```shell
Iron Fish version            0.1.36 @ 6dfc736
Iron Fish library            0.0.14 @ 6dfc736
Operating system             Windows_NT x64
CPU model(s)                 AMD Ryzen 9 5950X 16-Core Processor
CPU threads                  32
RAM total                    31.92 GiB
Node version                 v16.15.0
ironfish in PATH             true
Telemetry enabled            true
Accounts head hash           000000000003df967a30cee4e86e225b87a9bcb513359b0bd47f4278d1604246
Accounts head in chain       true
Accounts head sequence       48017
```

### Relevant log output

```shell
C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 5.18354554 ($ORE 518,354,554)
Amount available to spend: $IRON 5.18354554 ($ORE 518,354,554)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 5.16682201 ($ORE 516,682,201)
Amount available to spend: $IRON 5.16682201 ($ORE 516,682,201)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 5.16682201 ($ORE 516,682,201)
Amount available to spend: $IRON 5.16682201 ($ORE 516,682,201)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 5.06682200 ($ORE 506,682,200)
Amount available to spend: $IRON 5.06682200 ($ORE 506,682,200)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 5.06682200 ($ORE 506,682,200)
Amount available to spend: $IRON 5.06682200 ($ORE 506,682,200)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 4.78354550 ($ORE 478,354,550)
Amount available to spend: $IRON 4.78354550 ($ORE 478,354,550)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 4.68354549 ($ORE 468,354,549)
Amount available to spend: $IRON 4.68354549 ($ORE 468,354,549)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 4.62973164 ($ORE 462,973,164)
Amount available to spend: $IRON 4.62973164 ($ORE 462,973,164)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 4.62973164 ($ORE 462,973,164)
Amount available to spend: $IRON 4.62973164 ($ORE 462,973,164)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 4.57706473 ($ORE 457,706,473)
Amount available to spend: $IRON 4.57706473 ($ORE 457,706,473)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 4.48509192 ($ORE 448,509,192)
Amount available to spend: $IRON 4.48509192 ($ORE 448,509,192)

C:\Users\Mark>ironfish accounts:balance msa6867
Account - msa6867 (2c47304)

The balance is: $IRON 4.28354545 ($ORE 428,354,545)
Amount available to spend: $IRON 4.28354545 ($ORE 428,354,545)
```

### Graffiti

msa6867

**Comment by lwisne** [comment]
This is somewhat hard to follow - it is not clear what operations were happening when. It is possible you may be misunderstanding what the "correct" value should be, would like to double check that.

**Comment by hughy** [comment]
Closed in favor of duplicate issue in #1639
